src/main/java/games/strategy/engine/random/PropertiesDiceRoller.java

```diff
@@ -145,7 +145,7 @@ public String postRequest(final int max, final int numDice, final String subject
       httpPost.addHeader("X-Triplea-Game-UUID", gameUuid);
       final String host = m_props.getProperty("host");
       final int port = Integer.parseInt(m_props.getProperty("port", "80"));
```

Comment:
An alternative solution is to change the default port to 443 and remove the explicit `port` property from _tripleawarclub.properties_.

That would also make it consistent with the `HttpHost` definition below.  As it is, this change always requires an HTTPS endpoint due to the hard-coded `"https"` scheme.  So we might as well make the default port the default HTTPS port instead of the default HTTP port.

Reply:
Makes sense. I removed the port from the property file and updated the port in java config:  cff5b34